### PR TITLE
Update SSH Access Process diagram

### PR DESCRIPTION
**Pull request by stone-z**
Towards https://github.com/giantswarm/giantswarm/issues/15422 - updates the terminology used in the "SSH Access Process" diagram and removes image text. The source for the image used here is in [this Google Slides project](https://docs.google.com/presentation/d/1IjaJyK2rLzCd3eb_Tv6c6B7ID4e44IBJDXxfU8pN_00/).

I also took the opportunity to touch up the Secure access page

### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Comment by stone-z** [comment]
I'm not sure how to preview this change. This is the new image in PNG form for the sake of this review:
SSH Access process

**Comment by MarcelMue** [comment]
Maybe I am misunderstanding, but isn't the bastion missing here? IIRC we go through that as well

Or is that irrelevant here

**Comment by stone-z** [comment]
This is more about the auth side. There's another image with the actual infrastructure
https://docs.giantswarm.io/security/cluster-access/

File: src/content/security/cluster-access/index.md [hunk 1]
```diff
@@ -38,32 +39,32 @@ If you would like to know more about the different parts of the Giant Swarm infr
...
 ## User access
 
-User access is limited to the offered APIs for interaction with your clusters.
+User access is provided via two APIs for interaction with your clusters.
 
 ### Giant Swarm API
 
 High-level cluster management access is provided to you via the Giant Swarm API. This includes creating, scaling, and deleting your clusters, as well as other organization and user management functions.
-Network access to the API endpoint is usually whitelisted to a certain range of IP addresses. It can also be configured to work over a VPN following the general VPN connection schema shown below under [admin access](#admin-access). In this case the connection to the API residing in the cluster can be established only via your configured VPN.
+Network access to the API endpoint is typically whitelisted to a certain range of IP addresses. It can also be configured to work over a Virtual Private Network (VPN) following the general VPN connection schema shown below under [admin access](#admin-access). In this case, the connection to the API residing in the cluster can be established only via your configured VPN.
```
Comment (marians) [inline src/content/security/cluster-access/index.md:47]: Can `virtual private network` be spelled in lowercase? Title case makes it look like some product name.

Unrelated to your changes, but can we find a replacement term for `whitelisted` here, too?

File: src/content/security/cluster-access/index.md [hunk 2]
```diff
@@ -38,32 +39,32 @@ If you would like to know more about the different parts of the Giant Swarm infr
...
 
 ### Kubernetes API
 
-The Kubernetes API of each cluster is exposed to customers. You have full control over the users that are created via the Giant Swarm API. Additionally, you can also manage users by connecting an external Identity Provider to the Kubernetes API.
+The Kubernetes API of each cluster is exposed to customers. Authorized users of the cluster can be managed by connecting an external Identity Provider to the Kubernetes API, or by provisioning them directly through the Giant Swarm API. In either case, you retain full control over user management.
```
Comment (marians) [inline src/content/security/cluster-access/index.md:51]: Like above, can `Identity Provider` be spelled `identity provider`?

**Comment by marians** [comment]
Looks like `ssh_access_process_transparent.svg` is not used.

**Comment by marians** [comment]
Since you seem to give this article a full review, please update `last_review_date: 2020-01-21` to the current date.

**Comment by stone-z** [comment]
I just included the transparent SVG as an option for the future. Other feedback has been incorporated

**Review by MarcelMue — APPROVED** [review]

**Review by marians — APPROVED** [review]

**Pull request merged**